.github/workflows/main.yml

```diff
@@ -75,7 +75,7 @@ jobs:
 
       - name: Maven Build and Test
         shell: bash
-        run: mvn -DnoAcceptanceTests=true install ${{ env.MAVEN_CLI_OPTS}}
+        run: mvn -DnoAcceptanceTests=true -DunitTests.parallel=true -T 1C install ${{ env.MAVEN_CLI_OPTS}}
```

Comment:
I know that this was not introduced by this PR,  but what do you think about renaming `noAcceptanceTests` to `skipAcceptanceTests`?

Reply:
Done, thank you.